warehouse/models/staging/gtfs/stg_gtfs_schedule__fare_rules.sql

```diff
@@ -0,0 +1,22 @@
+WITH external_fare_rules AS (
+    SELECT *
+    FROM {{ source('external_gtfs_schedule', 'fare_rules') }}
+),
+
+stg_gtfs_schedule__fare_rules AS (
+
+    -- Trim all string fields
+    -- Incoming schema explicitly defined in gtfs_schedule_history external table definition
+    -- select distinct because of several instances of full duplicates, ex. ITP ID 294, URL 1 on 2022-03-09
```

Comment:
Same Q as above -- can we create an intermediate table and dedupe by freq here instead of using `SELECT DISTINCT`?